login.html

```diff
@@ -29,12 +29,14 @@
       <form class="email-box">
         <label class="email-label" for ="email">이메일</label>
         <input class="input" placeholder="이메일을 입력해주세요" id="email">
+        <div class="error-message"></div>
       </form>
       <form class="pw-box">
         <label class="pw-label" for ="pw">비밀번호</label>
         <input class="input" placeholder="비밀번호를 입력해주세요" type="password" id="pw">
+        <div class='error-message-pw'></div>
       </form>
-      <button class="login-btn" onclick="location.href='/login'">로그인</button>
+      <button id="login" class="login-btn" onclick="location.href='/items'">로그인</button>
```

Comment:
P3:
해당 button은 disabled true로 해두는 것이 동작상 좋지 않을까요? 🧐
처음에는 input들이 비어있어서 invalid할테니까용

```suggestion
      <button id="login" disabled class="login-btn" onclick="location.href='/items'">로그인</button>
```